modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerScaleUpTest.java

```diff
@@ -102,44 +99,32 @@ void testDataNodesPropagationAfterScaleUpTriggered() throws Exception {
 
         assertLogicalTopology(clusterNodes2, keyValueStorage);
 
-        distributionZoneManager.createZone(
-                new DistributionZoneConfigurationParameters.Builder(ZONE_1_NAME)
-                        .dataNodesAutoAdjustScaleUp(IMMEDIATE_TIMER_VALUE)
-                        .dataNodesAutoAdjustScaleDown(INFINITE_TIMER_VALUE)
-                        .build()
-        ).get();
+        createZone(ZONE_NAME, IMMEDIATE_TIMER_VALUE, INFINITE_TIMER_VALUE, null);
 
-        assertDataNodesForZone(ZONE_1_ID, clusterNodes2, keyValueStorage);
+        assertDataNodesForZone(ZONE_ID, clusterNodes2, keyValueStorage);
 
         topology.putNode(NODE_3);
 
         Set<LogicalNode> clusterNodes3 = Set.of(NODE_1, NODE_2, NODE_3);
 
         assertLogicalTopology(clusterNodes3, keyValueStorage);
 
-        assertDataNodesForZone(ZONE_1_ID, clusterNodes3, keyValueStorage);
+        assertDataNodesForZone(ZONE_ID, clusterNodes3, keyValueStorage);
         assertDataNodesForZone(DEFAULT_ZONE_ID, clusterNodes3, keyValueStorage);
     }
 
     @Test
     void testDataNodesPropagationAfterScaleUpTriggeredOnNewCluster() throws Exception {
         startDistributionZoneManager();
 
-        distributionZoneManager.alterZone(DEFAULT_ZONE_NAME,
-                new DistributionZoneConfigurationParameters.Builder(DEFAULT_ZONE_NAME)
-                        .dataNodesAutoAdjustScaleUp(IMMEDIATE_TIMER_VALUE)
-                        .dataNodesAutoAdjustScaleDown(INFINITE_TIMER_VALUE)
-                        .build()
-        ).get();
+        alterZone(DEFAULT_ZONE_NAME, IMMEDIATE_TIMER_VALUE, INFINITE_TIMER_VALUE, null);
 
-        distributionZoneManager.createZone(
-                new DistributionZoneConfigurationParameters.Builder(ZONE_1_NAME).dataNodesAutoAdjustScaleUp(IMMEDIATE_TIMER_VALUE).build()
-        ).get();
+        createZone(ZONE_NAME, IMMEDIATE_TIMER_VALUE, null, null);
```

Comment:
Why do you think that such shortcut worth writing it?  Original option is much more readable.